hera_mc/mc_session.py

```diff
@@ -723,3 +724,178 @@ def get_paper_temps(self, starttime, stoptime=None):
 
         return self._time_filter(PaperTemperatures, 'time', starttime,
                                  stoptime=stoptime)
+
+    def add_ant_metric(self, obsid, ant, pol, metric, val):
+        """
+        Add a new antenna metric to the M&C database.
+
+        Parameters:
+        ------------
+        obsid: long integer
+            observation identification number
+        ant: integer
+            antenna number
```

Comment:
So there's going to be a correlator input string (not number) in the files the correlator writes in the future and I do think we want to persist it (this will require some development work on pyuvdata, but it should be pretty easy). There will also soon be code to get the correlator input string for an obsid + antenna, but it's still being written. So soonish we'll want to add that, but I'll leave it up to you @adampbeardsley whether to do it in this PR or add it later.